app/models/early_career_payments/dqt_record.rb

```diff
@@ -36,16 +36,16 @@ def eligible_induction?
 
     # TODO: May need to prioritise subject chosen by highest award amount?
     def eligible_itt_subject_for_claim
-      return nil if !itt_subjects || itt_subjects.empty?
+      raise if !itt_subject_groups || itt_subject_groups.empty?
```

Comment:
Now that you've removed the `find` after creating the array in `itt_subject_groups`, I think this can be reduced to

```
raise if itt_subject_groups.empty?
```